spec/system/candidate_interface/candidate_entering_degrees_spec.rb

```diff
@@ -287,7 +287,10 @@ def when_i_click_to_change_my_undergraduate_degree_type
   end
 
   def when_i_click_to_change_my_undergraduate_degree_year
-    click_change_link('year')
+    start_year_row = find('.govuk-summary-list__row', text: 'Start year')
+    within start_year_row do
+      click_change_link('year')
+    end
```

Comment:
:+1: Made me wonder, if this is a common pattern, could you go further with a helper that looks something like: `click_change_link('year', within: 'Start year')`? Maybe it's rare enough that it doesn't make sense.

Reply:
Like that idea. Tested it by going through and using this helper across the candidate interface specs. I found the need to scope wasn't that common (and in a few cases, it was just as clear to leave the `within` call in the spec itself). 

Pushed those replacements in the newest commit. WDYT?

Reply:
Looks good to me, I don't think it's worth adding the extra complexity to the helper for a rare case.